### PR TITLE
Add VLAN enhancement documentation

### PR DESCRIPTION
**Pull request by yaocw2020**
Related issue: https://github.com/harvester/harvester/issues/2735

### CODE REVIEW

**Review by vickyhella — APPROVED** [review]
Other than a few nits, LGTM!

File: docs/networking/clusternetwork.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- NetworkConfig
+- Network
+---
+# Harvester Networking
+
+## Concepts
+
+### Cluster Network
+
+We bring in a new concept called cluster network for traffic isolation.
```
Comment (guangbochen) [inline docs/networking/clusternetwork.md:18]:
```suggestion
_Available as of v1.1.0_

In Harvester v1.1.0, we introduced a new concept called cluster network for traffic isolation.
```

File: docs/networking/clusternetwork.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+The following diagram describes a typical network architecture that separates data-centre (DC) traffic from out-of-band(OOB) traffic.
+![](/img/v1.1/networking/traffic-isolation.png)
+
+We abstract the sum of devices, links, and configurations on a traffic-isolated forwarding path on the Harvester as a cluster network.
+
+In the above case, there will be two cluster networks corresponding to two traffic-isolated forwarding paths.
+
+### Network Configuration
+
+The specifications including network devices of the Harvester hosts can be different. To be compatible with such a heterogeneous cluster, we design network configuration.
```
Comment (guangbochen) [inline docs/networking/clusternetwork.md:29]:
```suggestion
Specifications including network devices of the Harvester hosts can be different. To be compatible with such a heterogeneous cluster, we designed the network configuration.
```

File: docs/networking/clusternetwork.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+The network configuration can only work under a certain cluster network. Each network configuration corresponds to a set of hosts with uniform network specifications. Therefore, it requires multiple network configurations to enable a cluster network on multiple non-uniform hosts.
```
Comment (guangbochen) [inline docs/networking/clusternetwork.md:31]:
```suggestion
Network configuration only works under a certain cluster network. Each network configuration corresponds to a set of hosts with uniform network specifications. Therefore, multiple network configurations are required for a cluster network on non-uniform hosts.
```

File: docs/networking/clusternetwork.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+### Network
+
+The network is an interface used in VMs to connect with the host network. Same with network configuration, every network must be under a certain cluster network except the built-in network.
```
Comment (guangbochen) [inline docs/networking/clusternetwork.md:35]:
```suggestion
A network is an interface in a virtual machine that connects to the host network. As with network configuration, every network except the built-in `mgmt` network must be under some cluster network.
```

File: docs/networking/harvester-network.md [hunk 5]
```diff
@@ -1,127 +1,64 @@
...
- 
-   ![create-vlan-network.png](/img/v1.1/networking/create-network.png)
-  
- 2. Configure a route in order to allow the hosts to connect to the VLAN network using IPv4 addresses. The CIDR and gateway of the VLAN network are mandatory parameters for the route configuration.  You can configure the route by choosing one of the following options:
-    - auto(DHCP) mode: the Harvester network controller will get the CIDR and gateway values from the DHCP server using the DHCP protocol. Optionally, you can specify the DHCP server address.
-      ![](/img/v1.1/networking/create-network-auto.png)
-    - manual mode: You need to specify the CIDR and gateway values manually.
-      ![](/img/v1.1/networking/create-network-manual.png)
+2. Configure a route to allow the hosts to connect to the VLAN network using IPv4 addresses. The CIDR and gateway of the VLAN network are mandatory parameters for the route configuration.  You can configure the route by choosing one of the following options:
+- auto(DHCP) mode: the Harvester network controller will get the CIDR and gateway values from the DHCP server using the DHCP protocol. Optionally, you can specify the DHCP server address.
```
[the remark this answers is not in the document]
Comment (guangbochen) [reply docs/networking/harvester-network.md:42]: also, can we fix the indent? thanks

File: docs/networking/clusternetwork.md [hunk 6]
```diff
@@ -14,62 +14,63 @@ keywords:
...
+Specifications including network devices of the Harvester hosts can be different. To be compatible with such a heterogeneous cluster, we designed the network configuration.
 
-The network configuration can only work under a certain cluster network. Each network configuration corresponds to a set of hosts with uniform network specifications. Therefore, it requires multiple network configurations to enable a cluster network on multiple non-uniform hosts.
+Network configuration only works under a certain cluster network. Each network configuration corresponds to a set of hosts with uniform network specifications. Therefore, multiple network configurations are required for a cluster network on non-uniform hosts.
 
 ### Network
 
-The network is an interface used in VMs to connect with the host network. Same with network configuration, every network must be under a certain cluster network except the built-in network.
+A network is an interface in a virtual machine that connects to the host network. As with network configuration, every network except the built-in `mgmt` network must be under some cluster network.
```
Comment (vickyhella) [inline docs/networking/clusternetwork.md:36]:
```suggestion
A network is an interface in a virtual machine that connects to the host network. Same as network configuration, every network (except the built-in `mgmt` network) must be under some cluster network.
```

Comment (vickyhella) [reply docs/networking/clusternetwork.md:36]: Does it mean `same as`?

**Review by guangbochen — APPROVED** [review]

**Review by bk201 — APPROVED** [review]
LGTM

**Pull request merged**